src/zenml/cli/server.py

```diff
@@ -128,6 +85,12 @@ def get_active_deployment(local: bool = False) -> Optional["ServerDeployment"]:
     help="Use a custom Docker image for the ZenML server. Only used when "
     "`--docker` is set.",
 )
+@click.option(
+    "--ngrok",
+    type=str,
+    default=None,
+    help="Specify an ngrok auth token to use for exposing the ZenML server.",
+)
```

Comment:
How about we call this auth-token or something more obvious to the user?

Reply:
Makes sense I renamed this to `ngrok-token` and also renamed `ngrok` args to `ngrok_token` everywhere